web/shared-beans/src/main/java/io/deephaven/web/shared/data/Range.java

```diff
@@ -15,15 +14,7 @@ public class Range implements Comparable<Range> {
     private final long first;
     private final long last;
 
-    // serialization
-    Range() {
-        this(0, 0);
-    }
-
     public Range(long first, long last) {
-        if (first > last) {
-            throw new IllegalStateException(first + " > " + last);
-        }
```

Comment:
Why was this check removed? Seems this assertion would still be applicable, `Range` assumes that `first <= last` in a few places in the code.